usage/jsgui/src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -777,14 +777,22 @@ define([
             }
         },
         getConfigMap:function() {
-            var map = {}
+            var that = this;
+            var map = {};
             $('.app-add-wizard-config-entry').each( function (index,elt) {
                 map[$('#key',elt).val()] = 
                     $('#checkboxValue',elt).length ? $('#checkboxValue',elt).is(':checked') :
-                    $('#value',elt).val()
+                    that.getConvertedConfigValue($('#value',elt).val())
```

Comment:
Suggest hoisting `getConvertedConfigValue` out of this view to be a separate function.